src/CatalystClient.ts

```diff
@@ -53,12 +53,12 @@ export class CatalystClient implements CatalystAPI {
         return this.contentClient.fetchStatus(options)
     }
 
-    fetchAllDeployments<T extends DeploymentBase = DeploymentWithMetadataContentAndPointers>(filters?: DeploymentFilters, fields?: DeploymentFields<T>, options?: RequestOptions): Promise<T[]> {
-        return this.contentClient.fetchAllDeployments(filters, fields, options)
+    fetchAllDeployments<T extends DeploymentBase = DeploymentWithMetadataContentAndPointers>(filters?: DeploymentFilters, sort?: DeploymentSorting, fields?: DeploymentFields<T>, options?: RequestOptions): Promise<T[]> {
```

Comment:
It looks like we are making a breaking change here. Since we are going to do so, could we use make the parameters be an object, like on the content server? Same with `streamAllDeployments`